spec/requests/exception_hunter/errors_request_spec.rb

```diff
@@ -1,6 +1,26 @@
 require 'rails_helper'
 
 describe 'Errors', type: :request do
+  describe 'show' do
+    let(:last_error) { ExceptionHunter::Error.last.id }
+
+    subject { get "/exception_hunter/errors/#{last_error}" }
```

Comment:
```suggestion
    subject { get "/exception_hunter/errors/#{last_error.id}" }
```

Reply:
Wasn't this supposed to take a group error id?

Reply:
lol yes